platformio.ini

```diff
@@ -19,21 +19,30 @@
 ; mesmerizer    HUB75 info panel with audio effects, weather, info, etc,
 
 [platformio]
-default_envs = 
+default_envs = mesmerizer
 
 ; ==================
 ; Base configuration
 ;
 ; Options that are used (or extended) in all device sections (and hence environments) are defined here
 
 [base]
-upload_port     = 
-monitor_port    = 
+upload_port     = /dev/cu.usbserial-8411301
+monitor_port    = /dev/cu.usbserial-8411301
```

Comment:
I think we really should clear default_envs, upload_port and monitor_port.

Reply:
I think it should be done automatically to compensate for human frailties.  DIdn't we already strip the env?  Can't we do all three?

Reply:
In CI we do clear default_envs and we could clear the ports, yes. For CI we don't even have to clear the ports, because they're not used in the CI flow. Any changes CI makes to the code it pulls from the repo are not persisted in the codebase though, so the code in the repo is and remains what you (the contributor) push to it.

I'm sure it would be possible to put a GitHub workflow in place that follows up a push to the code with an additional commit that removes or replaces certain lines, but that's a form of automation I'm not yet familiar with. There are some obvious pitfalls there; for one you wouldn't want the automated commit to trigger an endless loop of commits. I'm sure there are ways to navigate around that though.
However, it also just feels weird to outdate the local tree of a contributor the same second they push to the repo - which is what an automated commit like this would do.

Another option would be to use the same approach with platformio.ini as we did with secrets.h. That is, add platformio.ini to .gitignore and include a platformio.ini "template" instead. The issue with that is that the local platformio.ini gets out of sync "without warning" whenever the template is updated.